parts/k8s/kubernetesconfigs.sh

```diff
@@ -185,8 +185,10 @@ function configureCNI() {
     retrycmd_if_failure 30 6 10 modprobe br_netfilter || exit $ERR_MODPROBE_FAIL
     echo -n "br_netfilter" > /etc/modules-load.d/br_netfilter.conf
     if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
-        mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
-        chmod 600 $CNI_CONFIG_DIR/10-azure.conflist
+	# Casey TODO: When using Calico with Azure networking, we don't want this CNI
```

Comment:
This is one area where I'm not sure what the best approach is - when using Calico for network policy with the Azure plugin, Calico will install its own config via the DaemonSet, so we don't want to install this one.

I don't see a NETWORK_POLICY environment variable set in this script, though I could be missing something?

Reply:
@jackfrancis I'd appreciate any thoughts on how to handle this bit of code.

We need skip installation of this file when using NetworkPolicy=Calico with NetworkPlugin=Azure.

But right now, this always gets installed when NetworkPlugin=Azure. Is there recommended way for me to plumb the right information to this stage?

Reply:
@caseydavenport NETWORK_POLICY could be added in kubernetesmastervars.t, e.g. `' NETWORK_POLICY=',parameters('networkPolicy')`